src/styles/_header.scss

```
@@ -25,6 +25,11 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
+    transition: transform, $transition-dur, color, $transition-dur;
```

You don't have transform here.
```suggestion
    transition: color, $transition-dur;
```